backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+import numpy as np
+
+from nodes.properties.inputs import BoolInput, ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+
+from .. import adjustments_group
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:log2lin",
+    description="Convert all channels in an image to a scene linear encoding using the Cineon logarithmic function set.",
+    name="Log2Lin",
```

Comment:
I think we should call this node "Log To Linear" to be consistent with names of other nodes (e.g. "Metal To Specular", "ONNX To NCNN", "Text As Image").

Reply:
Elaborating on this note, I have a more general question about the naming schema for nodes: I noticed a lot of the nodes are also using additional verbs such as "Change", "Create", "Add", etc. Is there some sort of specific thinking around this approach so far? Or is it solely based on providing clarity to the Nodes function?

Reply:
We don't have any kind of consistent system right now, we just do whatever seems right at the time. We've never really gone back and reviewed anything or tried to make it consistent

Reply:
Node renamed and updated explanations and other items. Relinted. Gtg from my pov.